daemon/src/maker_cfd.rs

```diff
@@ -290,7 +290,7 @@ impl Actor {
         .unwrap();
 
         self.takers
-            .do_send_async(maker_inc_connections::TakerMessage {
+            .send(maker_inc_connections::TakerMessage {
```

Comment:
Could you maybe add a comment to this `send` so we are aware that it should not be changed to be async?